epoch.go

```diff
@@ -897,12 +1045,39 @@ func (e *Epoch) locateBlock(seq uint64, digest []byte) (Block, bool) {
 	return nil, false
 }
 
-func (e *Epoch) proposeBlock() error {
-	block, ok := e.BlockBuilder.BuildBlock(e.finishCtx, e.Metadata())
-	if !ok {
-		return errors.New("failed to build block")
+func (e *Epoch) buildBlock() {
```

Comment:
Is it ever possible for `canBeImmediatelyVerified` to be `false` here?

Reply:
No, I probably should change it to true.

Reply:
changed it to always be [true](https://github.com/ava-labs/Simplex/pull/38/commits/fc64e39e358b24873f65f10db14f39127f3475ce).